src/platform/forms-system/src/js/review/ReviewFieldTemplate.jsx

```diff
@@ -1,26 +1,28 @@
 import React from 'react';
+import { getNestedUISchema } from '../helpers';
+
 /*
  * This is the template for each field (which in the schema library means label + widget)
  */
 
 export default function ReviewFieldTemplate(props) {
-  const { children, uiSchema, schema } = props;
+  const { children, uiSchema: startingUISchema, schema } = props;
+  const uiSchema = getNestedUISchema(startingUISchema);
   const label = uiSchema['ui:title'] || props.label;
   const description = uiSchema['ui:description'];
   const textDescription = typeof description === 'string' ? description : null;
   const DescriptionField =
     typeof description === 'function' ? uiSchema['ui:description'] : null;
 
+  if (uiSchema?.['ui:reviewField']) {
+    return uiSchema['ui:reviewField'](props);
+  }
```

Comment:
We need to ensure this doesn't break other forms. Moving this logic above the following statement means that arrays or objects with a `ui:reviewField` will render differently.

I'm not comfortable with the level of testing we have on the review page right now to trust our CI to catch any breaking changes this might introduce.